inc/sainexthopgroup.h

```diff
@@ -133,6 +142,33 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Configured group size
+     *
+     * Maximum desired number of members. The real size should
+     * be queried from SAI_NEXT_HOP_GROUP_ATTR_ACTUAL_SIZE
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_ONLY
+     * @validonly SAI_NEXT_HOP_GROUP_ATTR_TYPE == SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP
+     * @isresourcetype true
```

Comment:
Can you explain the intention here for marking as resource -
Is it querying a specific next hop group object to see how many entries are still available ? 
or is the intention to query how many available next hop groups of a certain configured size (and optionally also type) can still be created ?

Reply:
To query how many NH groups of a certain size can be crated.